CRM/Contribute/Form/Contribution.php

```diff
@@ -1838,6 +1882,26 @@ protected function invoicingPostProcessHook($submittedValues, $action, $lineItem
   }
 
   /**
+   * Get last payment for contribution
```

Comment:
@pradpnayak please reply to @eileenmcnaughton indicating that you have accepted her point and will make a change or why you don't think it is necessary.